### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by ygoyal18**
Follows - https://github.com/armPelionEdge/meta-pelion-edge/pull/148

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,3 @@
+mbed_cloud_dev_credentials.c
+update_default_resources.c
+upgradeCA.cert
```
Comment (costanic) [inline .gitignore:3]: I think this line can be deleted since we're not doing verified boot anymore.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # Build environment for Pelion Edge firmware
 
-This repo contains a set of instructions (this README) and some scripts for building the Pelion OS Edge firmware image.
+This repo contains a set of instructions (this README) and some scripts for building the Pelion Edge firmware image.
+When used in conjuction with the [Repo Manifest](https://github.com/armpelionedge/manifest-pelion-edge) repository, it automates the complete [Yocto build instructions](https://github.com/armpelionedge/meta-pelion-edge/blob/dev/BUILD.md).
```
Comment (costanic) [inline README.md:4]: this points to dev branch on the manifest repo.  should this point to master instead?

File: README.md [hunk 3]
```diff
@@ -15,8 +16,17 @@ If you cloned the poky repo at a different location, or named the repo a differe
     docker
     build-essential
 
+   [Docker install instructions](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+
 ## Easy Instructions
 
+### Credentials
+   Several sets of credentials are needed to build the firware image. If they are present in this directory, the Makefile will copy them to the appropriate location for the yocto build.  Please consult the [Yocto build instructions](https://github.com/armpelionedge/meta-pelion-edge/blob/dev/BUILD.md) for more information about these credentials.
```
Comment (costanic) [inline README.md:24]: this points to dev branch.

**Comment by ygoyal18** [comment]
Wait for #33, #34

**Pull request merged**